mysql-test/suite/innodb/t/scn_on_and_off.test

```diff
@@ -0,0 +1,54 @@
+--source include/have_innodb.inc
+
+--echo #
+--echo # Test different value of innodb_use_scn
+--echo #
+
+create table t1(id int, c1 int) engine=innodb;
+insert into t1 value(1,1);
+insert into t1 value(2,1);
+insert into t1 value(3,1);
+update t1 set c1=2 where id=1;
+update t1 set c1=2 where id=2;
+check table t1;
+--source include/wait_all_purged.inc
+
+--source include/shutdown_mysqld.inc
+--let $restart_parameters= --innodb-use-scn=ON
+--source include/start_mysqld.inc
+show variables like '%use_scn%';
+
+insert into t1 values(4,1);
+update t1 set c1=2 where id=3;
+update t1 set c1=4 where id=3;
+check table t1;
```

Comment:
What would happen on `check table t1 extended`?

Reply:
Check table ... extended is not supported(`extended` is ignored and a warning is reported).